tools/jinja2-filters.py

```diff
@@ -0,0 +1,19 @@
+import base64
+
+class SecretIsNotString(Exception):
+    pass
+
+class NotSupportedDataType(Exception):
+    pass
+
+def filter_b64decode(secret_value, data_type = "string"):
+    if not isinstance(secret_value, str):
+        raise SecretIsNotString
+    if data_type == "string":
```

Comment:
It should be `if-elif-else`

Reply:
Does not really matter if the previouls `if` has a `raise` or `return`. For me it's fine either way.